specification/attributes/discount_handling.md

```diff
@@ -0,0 +1,49 @@
+
+Amortization is a process used to break down and spread purchase costs over a period of time or term of use. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on the resource's usage, attributing the prorated cost for each unit of billing. Amortization enables users of billing data to distribute purchase charges to the appropriate audience in support of cost allocation efforts. Discount Handling for purchased commitments is commonly used for scenarios like calculating utilization and implementing chargeback for the purchase amount.
+
+While providers may use different terms to describe discounts, FOCUS identifies a discount as being a reduced price applied directly to a row. Any price or cost reductions that are awarded after the fact are identified as a "Credit" Charge Subcategory. One example might be when a provider offers a reduced rate after passing a certain threshold of usage or spend.
+
+All rows defined in FOCUS MUST follow the discount handling requirements listed below.
+
+## Attribute ID
+
+DiscountHandling
+
+## Attribute name
+
+Discount Handling
+
+## Description
+
+Indicates how to include and apply discounts to usage charges or rows.
+
+## Requirements
+
+* All applicable discounts SHOULD be applied to each row they pertain to and SHOULD NOT be negated in a separate row.
+* All discounts applied to a row MUST apply to the entire charge.
+  * Multiple discounts MAY apply to a row, but they MUST apply to the entire charge covered by that row.
+  * If a discount only applies to a portion of a charge, then the discounted portion of the charge MUST be split into a separate row.
+  * Each discount MUST be identifiable using existing FOCUS columns.
+    * Rows with a commitment-based discount applied to it MUST include a CommitmentDiscountId.
+    * If a provider applies a discount that cannot be represented by a FOCUS column, they SHOULD include additional columns to identify the source of the discount.
+* ChargeSubCategory MUST NOT be null for rows where ChargeType is "Usage" and the row received reduced rates from a discount.
+* Purchased discounts (e.g., commitment-based discounts) MUST be amortized.
+  * The BilledCost MUST be 0 for any row where the commitment covers the entire cost for the charge period.
+  * The EffectiveCost MUST be the portion of the amortized purchase cost that applies to this row.
```

Comment:
```suggestion
  * The EffectiveCost MUST include the portion of the amortized purchase cost that applies to this row.
```

Currently reads like EffectiveCost must equal the amortised amount, which is not correct if there includes charges on the row itself.